test/common/quic/platform/quic_platform_test.cc

```diff
@@ -112,9 +118,14 @@ TEST_F(QuicPlatformTest, QuicExpectBug) {
   auto bug = [](const char* error_message) { QUIC_BUG(bug_id) << error_message; };
 
   auto peer_bug = [](const char* error_message) { QUIC_PEER_BUG(bug_id) << error_message; };
-
   EXPECT_QUIC_BUG(bug("bug one is expected"), "bug one");
   EXPECT_QUIC_BUG(bug("bug two is expected"), "bug two");
+#ifdef NDEBUG
+  // The 3rd triggering in release mode should be omitted.
```

Comment:
"should not be logged".  If there's a bug counter it should still be incremented.